src/app/organisms/noti-list/NotiList.tsx

```diff
@@ -105,7 +122,7 @@ function NotiList({
 }) {
   const mx = initMatrix.matrixClient;
 
-  const [notis] = useNotifications();
+  const [notis,,, nextToken, paginate, isLoading] = useNotifications();
```

Comment:
We should probably change it to an object lmao.

Reply:
yeah probably lol